p5.html.markdown

```diff
@@ -0,0 +1,55 @@
+---
+category: tool
+tool: p5
```

Comment:
This needs to be changed to
```
name: p5
```

Reply:
so, should I add that line
```
name : p5
```
??
Sorry for inconvenience, I am really new.

Reply:
I have added that line

Reply:
No issues with being new, we have all been there! I actually meant that you need to change the line `tool: p5` to `name: p5js`. Please make that change. Do let me know in case you need any help!